# controller gains (position + orientation)
float64[] Kp_gains
float64[] Kv_gains
# nullspace motion gains 
float64[] null_Kp_gains
float64[] null_Kv_gains
